pkg/logger/logger.go

```diff
@@ -22,10 +23,11 @@ var AllowedLevels = []string{"debug", "info", "warn", "error", "fatal", "panic",
 type CallerHook struct{}
 
 // Run adds additional context
-func (h CallerHook) Run(e *zerolog.Event, _ zerolog.Level, _ string) {
+func (h CallerHook) Run(e *zerolog.Event, lvl zerolog.Level, _ string) {
 	if _, file, line, ok := runtime.Caller(3); ok {
 		e.Str("file", fmt.Sprintf("%s:%d", path.Base(file), line))
 	}
+	metricsstore.DefaultMetricsStore.LogLvlCount.WithLabelValues(lvl.String()).Inc()
```

Comment:
I don't see why this metric is required. Counting number of logs  is not a useful metric. If the goal is to get metrics for the errors, then it needs to be done a lot differently - by inspecting the error log, and generating a metric with context, such as adding the error code information to the metric. 

If we really want to generate a generic metric for counting errors, then make the metric specific to error logs and inspect the level here for error. Counting `info/trace/debug` logs is totally unnecessary.

Reply:
@shashankram we will be adding a separate metric for a counter per Errcode

Reply:
@nshankar13 then this should be unnecessary.